commands/commands.go

```diff
@@ -60,6 +62,16 @@ var (
 // bootstrapCommands simply adds a given command to the allCommands array
 func bootstrapCommands(commands ...cli.Command) {
 	for _, command := range commands {
+		command.Before = func(c *cli.Context) error {
+			//Get access to the logger
+			configFile := c.String("config")
+			res := resources.InitResources(configFile)
+
+			res.Log.WithFields(log.Fields{
+				"Command arguments": c.Args(),
```

Comment:
Could you make it so it shows both flags and arguments? Basically, we want everything the user types on the command line after `rita`.  For instance, it would be very useful for debugging to know that a user used a custom config file and the `--reset` flag in the following command.

```
rita analyze --config etc/rita.yaml --reset vsagent2
```

But the above command produces the following log entry that only shows the dataset and not the extra flags.

```
time="2019-01-11T13:08:35-06:00" level=info msg="Running Command: analyze" Command arguments="[vsagent2]"
```

Reply:
I'll take a look at this later today. Thanks.

Reply:
Newest patch has the requested functionality, turns out it was simpler than I thought it was making it.

1. Just let me know if you want to workshop the format.
2. I added logging for global flags even though the project contains none, in case you add some in the future.